style-guide/how-to-article-guidelines.rst

```diff
@@ -0,0 +1,500 @@
+
+- Do *not* include explanatory or reference information in the action part of
+  a step. If needed, follow the step with one or more paragraphs that provide
+  such information.
+
+- Do *not* document system actions, responses, or results as steps. Put
+  necessary statements in paragraphs following the steps to which they apply.
+
+- Use screenshots sparingly. Screenshots can help to orient the user, but a
+  screenshot of every field or dialog box is usually not necessary.
+
+- To indicate that a step is optional, include *(Optional)*, in italics, as a
+  qualifier at the beginning of the step.
+
+- If more than one method exists for completing an action, document only one
+  method, usually the most efficient or preferred method.
+
+.. _consistent-text-format:
+
+Use consistent text formatting
+------------------------------
+
+Certain text should be formatted differently from the surrounding text to
+designate a special meaning or to make the text stand out to the user. Usually
+this formatting is accomplished by applying a different font treatment (such
+as bold, italics, or monospace).
+
+The following table covers the most common items that should be formatted. For
+more detailed formatting information, see the `Text formatting <https://developer.rackspace.com/docs/style-guide/style/text-formatting/>`__ section of
+the  style guide.
+
+.. list-table::
+   :widths: 30 20 50
+   :header-rows: 1
+
+   * - Text item
+     - Treatment
+     - Example
+   * - Text you want to emphasize
+     - Italics
+     - The offset *must* be a multiple of the limit (or zero).
+   * - Error messages
+     - Monospace
+     - ``The user does not have permission to perform this action.``
+   * - Code examples
+     - Monospace
+     - ``$ grep "ftp" /etc/xinetd.d/*``
+   * - File, directory, and folder names
+     - Bold
+     - However, if these items are shown as part of code, use monospace.
+       For example:
+       Copy the **index.php** file from your computer to the **content**
+       folder.
+       The following example shows a basic configuration for the FTP service,
+       in a file in the ```/etc/xinetd.d`` directory.
+   * - Keyboard key names and combinations
+     - Bold
+     - Press **Enter**.
+   * - UI fields
+     - Bold
+     - Select **Start > Control Panel**, and then click the **Mail** icon.
+   * - Placeholder text
+     - Show in camelCase.
+     - If the authoring tool allows it, apply italics to placeholders; if not,
+       enclose them in angle brackets.
+       For example: *hostName* or ``<hostName>``
+
+.. _clarify-pronouns:
+
+Clarify pronouns such as it, this, there, and that
+--------------------------------------------------
+
+Pronouns are useful, but you must ensure that their antecedents (the words
+that they are used in place of) are clear, and that they (the pronouns) don’t
+contribute to vagueness and ambiguity.
+
+- **It** - Ensure that the antecedent of *it* is clear. If multiple singular
```

Comment:
Did you want to use colons instead of the dashes for this list?

Reply:
Fixed